functions-iterators/starter.js

```diff
@@ -28,48 +28,157 @@ This will help us track your progress and it will give you practice writing good
 /* ++++====+++==++++==+++++======++++===++++++=====+++==++++++++++===== */
 
 // PART 0: Just DO it!!
-
-
+function justDoIt(name) {
+  typeof name == String
+return (` ${name} JUST DO IT`)
+}
 
 // PART 1: Big or Small String?
-
+function bigOrSmallString(word) {
+  typeof word == "string"
+  if ((word).length > 10 ){
+    return 'This word is loooooong!'
+  } else {
+   return 'This word is short 😬'
+  }
+}
 
 
 // PART 2: Odd or Even String Length?
 
-
+function oddOrEvenString (stringdete){
+ typeof stringdete == "string"
+  if ((stringdete).length % 2) {
+    return "This string's length is odd.."
+  }else {
+    return "This string length is TOTALLY even!"
+  }
+}
 
 // PART 3: Median
-
+function medianOfArray(array) { 
+var medianOfArray = 0, numsLen = array.length;
+array.sort(); 
+if (numsLen % 2 === 0)
+  {
+    medianOfArray = (array[numsLen / 2 - 1] + array[numsLen / 2]) /2
+  } else {
+    medianOfArray = array[(numsLen - 1) / 2];
+  }
+  return medianOfArray;
+}
 
 
 // PART 4: Sum Array
-
+function sumArray(arr) {
+ if(!Array.isArray(arr)) 
+ return;
+ let sum=0;
+ arr.forEach(each => {
+   sum+=each;
+
+ })
+ return sum
+}
 
 
 // PART 5: Vowel Count
-
+function vowelCount(str) {
+  const match = str.match(/[aeiou]/gi);
+  return match ? match.length : 0 ;
+}
 
 
 // PART 6: Initials
+function initials(personName) {
```

Comment:
this function will always reassign personName to your name what happens if I call this function with different name `initials('ghadeer nasser')`